install.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+# This script is meant to automate the deployment of Szurubooru via Docker/Docker-Compose,
+# and to automatically take the necessary steps to allow Szurubooru to run on ARM devices
+# such as the Raspberry Pi.
+
+###########################################[Notes]###########################################
+# * Written and tested on Raspberry Pi 4 running Raspbian 10 (armv7), heavily based on a fork
+#   by kaijuagenda for the Pi 3: https://github.com/kaijuagenda/szurubooru-rpi3
+#
+# * Modifies docker-compose.yml rather than relying on a static alternative file to hopefully
+#   avoid the need for parallel maintenance of two files.
+#############################################################################################
+
+# Check for root to ensure the accessibility of the Docker daemon, and to 
+# allow mount point permissions to be set after building.
+
+if [[ $EUID -ne 0 ]]; then
+    echo "Sorry, but this script needs root to properly function.  Please try again using sudo ./install.sh" 1>&2
```

Comment:
You don't always need root to run docker, see:

https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user

Reply:
Good point!  Removed Root check.